src/main/java/org/opentripplanner/framework/geometry/WgsCoordinate.java

```diff
@@ -155,6 +155,21 @@ public boolean equals(Object o) {
     return latitude == other.latitude && longitude == other.longitude;
   }
 
+  /**
+   * Return a new version of this coordinate where latitude/longitude are rounded to 4 decimal
+   * places which at the equator has ~11 meter precision.
+   * <p>
+   * See https://wiki.openstreetmap.org/wiki/Precision_of_coordinates
+   * <p>
+   * This is useful when you want to cache coordinate-based computations but don't need absolute
+   * precision. DO NOT USE THIS IN ROUTING (USE AT LEAST 7 DECIMALS)!
+   */
+  public WgsCoordinate roundToApproximate10m() {
+    var lat = DoubleUtils.roundTo4Decimals(latitude);
+    var lng = DoubleUtils.roundTo4Decimals(longitude);
+    return new WgsCoordinate(lat, lng);
+  }
+
```

Comment:
This method is inserted between hc() and eq(), please move it infront of toString(). See codestyle for order on methods.

Reply:
Done in https://github.com/opentripplanner/OpenTripPlanner/pull/4979/commits/0940da48bf46df3f3a0de55df51cfdbfeb249055